src/pipelines/sass.rs

```diff
@@ -59,17 +66,28 @@ impl Sass {
             anyhow!("error compiling sass for {:?}", &self.asset.path)
         })?;
 
-        // Hash the contents to generate a file name, and then write the contents to the dist dir.
-        let hash = seahash::hash(css.as_bytes());
-        let file_name = format!("{}-{:x}.css", &self.asset.file_stem.to_string_lossy(), hash);
-        let file_path = self.cfg.staging_dist.join(&file_name);
-        fs::write(&file_path, css).await.context("error writing SASS pipeline output")?;
+        // Check if the specified SASS/SCSS file should be inlined
+        let css_ref = if self.use_inline {
+            // Avoid writing any files, return the CSS as a String
+            CssRef::Inline(css)
+        } else {
+            // Hash the contents to generate a file name, and then write the contents to the dist dir.
+            let hash = seahash::hash(css.as_bytes());
+            let file_name = format!("{}-{:x}.css", &self.asset.file_stem.to_string_lossy(), hash);
+            let file_path = self.cfg.staging_dist.join(&file_name);
+
+            // Write the generated CSS to the filesystem
+            fs::write(&file_path, css).await.context("error writing SASS pipeline output")?;
+
+            // Generate a hashed reference to the new CSS file
+            CssRef::File(HashedFileOutput { hash, file_path, file_name })
+        };
```

Comment:
Solid approach! Great job. One thing: let's be consistent about using proper punctuation at the end of our comments.

Reply:
Thanks!

I just added them.